tcms/testruns/static/testruns/js/get.js

```diff
@@ -552,12 +579,17 @@ function renderTestExecutionRow(testExecution) {
         template.find('.comment-form').hide()
         return template
     }
+
     template.find('textarea')[0].id = `comment-for-testexecution-${testExecution.id}`
     template.find('input[type="file"]')[0].id = `file-upload-for-testexecution-${testExecution.id}`
 
+    // remove from expanded list b/c data may have changed
+    delete expandedExecutionIds[expandedExecutionIds.indexOf(testExecution.id)]
```

Comment:
I'm not sure why this is here, I don't remember adding it, and I think this is a rebase artifact. not sure if i should remove it

Reply:
This is rebase leftover. Was moved in 6135db37 10 lines above. Remove it.